ethereum_history_api/circuits/lib/src/rlp/decode.nr

```diff
@@ -146,6 +172,17 @@ pub fn get_small_string_offset_and_length(first_byte: u8) -> (u64, u64) {
     }
 }
 
+pub fn copy_subfragment<T, N, M>(data: Fragment<N, T>, offset: u64) -> Fragment<M, T> {
```

Comment:
This should be a function on fragment. I think - it exists

Reply:
Removed it with altered implementation of decode_list